### PR TITLE
Change internal container ports and update documentation

### PR DESCRIPTION
**Pull request by jpfleischer**
I am back with more container-internal port reconfigurations.. 😀

The intention is for the output to tell me straightaway the right port to connect to.
Also, README has been updated accordingly.

### CODE REVIEW

**Review by shankari — CHANGES_REQUESTED** [review]
Good catch! I am not sure why I had the server start at port 6060 and then remapped it in the docker-compose. I got this dockerfile originally from 
https://github.com/danjellesma/http-server/blob/master/Dockerfile#L7

https://github.com/e-mission/em-public-dashboard/commit/adc63eb58a449a32d0550bd613ead5e3a4b1189d

And then I changed it to 6060 (and then to 5050) but I didn't write down why.

This was all done in a scrambling hurry (we had MOUs in place and needed to get the deployments done), so it lives on as a cautionary tale around the importance of documenting the "why"

Please indicate "Testing done" before I merge.

**Comment by jpfleischer** [comment]
Testing is indeed done, as I expect to have the URLs working and the proper ports shown in the console once done building, and these are both true.

**Comment by jpfleischer** [comment]
Output has been added to previous issue comment.

File: README.md [hunk 1]
```diff
@@ -76,6 +70,14 @@ Note that this expects a standard setup with:
 
 When working with `docker compose`, it's generally recommended to avoid committing changes to the `docker-compose.dev.yml` file, especially if you're running the `./load_mongodump <dump tar>` script. This file is typically configured to work in a specific way for your development environment, and changes might not be applicable or useful for others working on the same project.
 
+If you feel the need to adjust the Dockerfiles or any scripts pertaining to the containers, you must rebuild the images with:
+
+```bash
+docker-compose -f docker-compose.dev.yml build
+```
```
Comment (shankari) [inline README.md:77]: This is not true. It is true for the production version which copies the files into the container, but not for the dev version, which just maps `viz_scripts` as a volume mount.

**Review by shankari — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jpfleischer) [reply README.md]: ok! I have added this clarification.

**Pull request closed without merging**